src/gt4py/next/iterator/transforms/collapse_tuple.py

```diff
@@ -105,6 +105,7 @@ def apply(
         ignore_tuple_size: bool = False,
         remove_letified_make_tuple_elements: bool = True,
         offset_provider=None,
+        is_local_view=None,
```

Comment:
I think I would prefer something like `within_stencil`, but feels smelly anyway.

Reply:
Makes sense and doesn't collide with the local view extreme of the IR vocabulary anymore. Changed here and for consistency in the cse.